File: LeetCode/Medium/1637-widest-vertical-area-between-two-points-containing-no-points/1637-widest-vertical-area-between-two-points-containing-no-points.java
class Solution {
    public int maxWidthOfVerticalArea(int[][] points) {
        int[] x = new int[points.length];

        for (int i = 0; i < points.length; ++i) x[i] = points[i][0];
        
        Arrays.sort(x);
        
        int ret = 0;
        
        for (int i = 1; i < x.length; ++i)
            ret = Math.max(ret, x[i] - x[i-1]);
        
        return ret;
    }
}

File: LeetCode/Medium/1637-widest-vertical-area-between-two-points-containing-no-points/1637-widest-vertical-area-between-two-points-containing-no-points.cpp
class Solution {
public:
    int maxWidthOfVerticalArea(vector<vector<int>>& points) {
        vector<int> x(points.size());
        
        for (int i = 0; i < x.size(); ++i) x[i] = points[i][0];
        
        sort(x.begin(), x.end());
        
        int ret = 0;
        
        for (int i = 1; i < x.size(); ++i) ret = max(ret, x[i] - x[i-1]);
        
        return ret;
    }
};